src/MergeContext.js

```diff
@@ -787,9 +787,24 @@ class PullRequest {
         return (this._rawPr.title + ' (#' + this._rawPr.number + ')' + '\n\n' + this._prBody()).trim();
     }
 
+    // Searches for the first non_ASCII_printable character and returns
+    // its position in the string.
+    _invalidCharacterPosition(str) {
+        const prohibitedCharacters = /[^\x20-\x7e]/; // allow non-special ASCII characters
+        const match = prohibitedCharacters.exec(str);
+        return match ? match.index : -1;
+    }
+
     _prMessageValid() {
         const lines = this._prMessage().split('\n');
-        for (let line of lines) {
+        for (let i = 0; i < lines.length; ++i) {
+            const line = lines[i];
+            // this._prBody() already cared about removing trailing \r characters
```

Comment:
```suggestion
            // this._prBody() already removed any trailing \r characters
```

Why are we talking about the _trailing_ `\r` here? A non-trailing `\r` in an `\r\n` sequence inside the text should also be ignored. Was it removed by the `.split()` call above?

Reply:
this_prBody() may have a multiline string and it removes any \r, substituting \r+\n with \n. Since split() here uses \n as a separator, we can mistakenly assume that a legitimate preceding \r would force _prMessageValid() return false. The comment reminds us of that. Also, any \r (without following \n) within the text (body or title, no matter) would be treated as a forbidden symbol and mean an invalid message. Is it wrong?

Reply:
Based on your description, the code works correctly, but the comment is lying -- `_prBody()` does not (and should not) remove trailing `\r` characters. I suggest adjusting the comment to better match reality:

```suggestion
            // _prBody() removed CRs in CRLF sequences; other CRs are banned
```

Reply:
I committed a better comment in 71def24.